### ISSUE TITLE
SCORN - FOV changes to really wide one when zooming with a weapon equipped

### ISSUE THREAD

**Issue by theGlaive** [RoseTheFlower/UltrawideIndex#30]
First time character acquires a weapon in the game (~30-40mins in) the fov breaks when looking down sight with or RMB. it chnges to really wide one.
it's happening even without this fix so it's the game itself.
any possibility to address it with this fix update?

*Sorry for double post. Accidently posted it in discussions first

**Comment by RoseTheFlower** [comment]
Hi, it would help to have a save file from just a bit prior to this. It being a potentially native issue may be a good sign meaning that it could get fixed in an official update.

**Comment by theGlaive** [comment]
Dang, I already moved on. I'll try to save one next time this issue appears

**Comment by theGlaive** [comment]
Can't find saves. I on game pass version through xbox app on windows.
this path doesn't have any saves: %LOCALAPPDATA%\Scorn\Saved\SaveGames\
I guess it for steam version only.
Any ideas where I can find saves for game pass version of the game?

**Comment by theGlaive** [comment]
for the default game I found workaround to enable window mode. then do alt+enter to maximize the window. this way weird fov change doesn't happen when aiming down sight.
downside of this is your tweak stops working in windowed mode.

**Comment by RoseTheFlower** [comment]
> Any ideas where I can find saves for game pass version of the game?

Most likely in the folders in \%LOCALAPPDATA%\Packages\KeplerInteractive.1439274AB3A46_ymj30pw7xe604\SystemAppData\wgs\

> downside of this is your tweak stops working in windowed mode.

It shouldn't be tied to any window mode, but I'm unable to fully reproduce because the choice of Windowed resolutions offered by the game is limited to 16:9 and similar resolutions.

**Comment by jzyinq** [comment]
Maybe this [workaround](https://steamcommunity.com/app/698670/discussions/1/5277681174356140003/#c5277681174359341415) will be helpful in finding final solution.

> There seems to be a fix (at least for me at 3440x1440): in video settings change to "Windowed" (not windowed fullscreen), then alt+enter to enter full screen. Now when aiming it keeps the same FOV.
> What I noticed is that doing this the way the game calculates FOV will change, so you'll probably neet to readjust the FOV in the settings to higher values.
> Hope this helps!

**Comment by RoseTheFlower** [comment]
That's probably the thing proposed by theGlaive as well, but now it makes sense to me. However, the fix still works on my end even when switching from Windowed via Alt+Enter.

Here are two images without and with the fix in windowed made fullscreen with Alt+Enter (FOV maxed out in both instances, I think):

![image](https://user-images.githubusercontent.com/4759227/195984882-841f31d8-7926-4fd1-b9a7-97eb293b87b4.png)
![image](https://user-images.githubusercontent.com/4759227/195984885-eeeff0ea-a96b-4b29-8947-b776d4a1ef34.png)

**Comment by theGlaive** [comment]
> That's probably the thing proposed by theGlaive as well, but now it makes sense to me. However, the fix still works on my end even when switching from Windowed via Alt+Enter.
> 
> Here are two images without and with the fix in windowed made fullscreen with Alt+Enter (FOV maxed out in both instances, I think):
> 
> ![image](https://user-images.githubusercontent.com/4759227/195984882-841f31d8-7926-4fd1-b9a7-97eb293b87b4.png) ![image](https://user-images.githubusercontent.com/4759227/195984885-eeeff0ea-a96b-4b29-8947-b776d4a1ef34.png)

When I was comparing my character had weapon in his hand and with your fix its nice and wide and i can see hands but after i did windowed/alt+enter the hands god cropped again like in default game. That’s why i though fix doesn’t work in this mode. But i’ll double check. Thanks Rose

**Comment by Funsponge** [comment]
I think you need to test somewhere you have the weapon equipped. I'm assuming the saves are somewhere in this folder, since there's not much of anything anywhere else?

%LOCALAPPDATA%\Packages\KeplerInteractive.1439274AB3A46_ymj30pw7xe604\SystemAppData\wgs\0009000002D614F1_000000000000000000000000736740B0

![image](https://user-images.githubusercontent.com/5205690/196292856-9ad0bd8a-edd6-45ed-b297-b2a52b2002de.png)

**Comment by RoseTheFlower** [comment]
Yes, I think it's a folder per slot but I may be wrong. The last modified and/or the biggest may be the save file to look for.

**Comment by Funsponge** [comment]
[KeplerInteractive.zip](https://github.com/RoseTheFlower/UltrawideIndex/files/9805173/KeplerInteractive.zip)

I've played for a few hours and unlocked a couple different weapons. Anecdotally, it seems to compound the FOV when you get to the locks that require the first weapon. It plays an animation when unlocking them, and the more times you do it the more extreme the FOV becomes.

Given the game is supposedly pretty short and we have a workaround by swapping back and forth to windowed mode, I'd say don't invest too much effort. Unless you're personally interested in playing through, I don't think you're likely to catch all these weird edge cases before you run out of game :-)

**Comment by Funsponge** [comment]
[6B8A43DE5F354ED08D806CAF704288AC.zip](https://github.com/RoseTheFlower/UltrawideIndex/files/9813970/6B8A43DE5F354ED08D806CAF704288AC.zip)

When I load this one I'm able to reproduce the issue every time by right clicking to aim.

![FOV_before](https://user-images.githubusercontent.com/5205690/196512080-0b0003da-cf68-4e41-a411-c90acbffbcf2.jpg)
![FOV_after](https://user-images.githubusercontent.com/5205690/196512103-570cb5b0-82fe-4020-a102-84a61db5fdac.jpg)

**Comment by Funsponge** [comment]
To follow up, I have now deleted your files and it still does crazy things when right clicking, so it would appear either my config is borked, or this is just a general game bug rather than a bug related to your modifications.

**Comment by RoseTheFlower** [comment]
Thanks. The config helped me find the culprit, which is your resolution being 1440p. I tested at 2560x1080 and 3840x1080 but it didn't happen at those, then it certainly does at 3440x1440. I'll see if I can find out more.

By the way, even in the OP it says that the issue is not specific to my fix, but I kept it open just in case I could help.

**Comment by RoseTheFlower** [comment]
Turns out that the bug does not affect 3440x1439 and 3440x1441 either, so creating and using either of these as a custom resolution could be another workaround. So far I'm not seeing anything in the code that would be causing this.
